### ISSUE TITLE
Workflow issue on submit data for unauthenticated user

### ISSUE THREAD

**Issue by cjhendrix** [OCHA-DAP/hdx-ckan#634]
Sign out.
Click `Submit Data`.
Sign in.
You are taken to `My Organizations` instead of the (desired) org select page (if they are Editor/Admin of an org) or the "you need to belong to an org" page (if they are not). 

I think this might be a heavy build (since it implies tracking where a user came from before login, which I think CKAN does not do).  If so, we may table it for later.

**Comment by danmihaila** [comment]
@cjhendrix can you still reproduce this one?

**Comment by cjhendrix** [comment]
Yes, it's still an issue.  But now we would expect the add data modal to open rather than the "select org" page.
